src/sass/layout/_navigation.sass

```diff
@@ -0,0 +1,27 @@
+.navigation
```

Comment:
Try to keep it simple. Don´t nest to much. Best practice are two levels deep. In worst case three levels deep.